platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/SwirldStateManagerImpl.java

```diff
@@ -121,8 +121,8 @@ public SwirldStateManagerImpl(
             @NonNull final PlatformContext platformContext,
             @NonNull final AddressBook addressBook,
             @NonNull final NodeId selfId,
-            @NonNull final PreConsensusSystemTransactionManager preConsensusSystemTransactionManager,
-            @NonNull final PostConsensusSystemTransactionManager postConsensusSystemTransactionManager,
+            @NonNull final PreconsensusSystemTransactionManager preConsensusSystemTransactionManager,
```

Comment:
change arg name to `preconsensusSystemTransactionManager`

Reply:
fixed